src/diffpy/utils/parsers/loaddata.py

```diff
@@ -26,6 +26,9 @@ def loadData(filename, minrows=10, **kwargs):
     filename -- name of the file we want to load data from.
     minrows  -- minimum number of rows in the first data block.
                 All rows must have the same number of floating point values.
+    headers  -- return also a dictionary of parameters specified in header
```

Comment:
can we have a bit more.  For example, this should be a bool and the behavior explained if it is set to False or True nad that the default is False.

Please revisit the other help statements (even the ones written by others) to try and make the clearer like that if htey need it.